### ISSUE TITLE
Editorialized Precompiler Descriptions

### ISSUE THREAD

**Issue by MulverineX** [lolgeny/awesome-mcc#10]
At the moment all of the precompiler descriptions are very short and vague, imo they should be edited to contain more distinguishing information.

**Comment by lolgeny** [comment]
Yeah I mostly copied what was on the github repos which weren't really that helpful, I agree
